### ISSUE TITLE
ppc: Error (shmem.c:165): Sysv shmem 0 with tail hole not supported

### ISSUE THREAD

**Issue by avagin** [xemul/criu#170]
```
======================== Run zdtm/transition/ipc in ns =========================
Start test
./ipc --pidfile=ipc.pid --outfile=ipc.out
Run criu dump
Run criu restore
=[log]=> dump/zdtm/transition/ipc/30/1/restore.log
------------------------ grep Error ------------------------
(00.624272)      5: Error (shmem.c:165): Sysv shmem 0 with tail hole not supported
(00.624749)      4: Error (cr-restore.c:952): 5 exited, status=1
(00.626993) Error (cr-restore.c:955): 54 killed by signal 9: Killed
(00.627200) Error (cr-restore.c:1832): Restoring FAILED.
------------------------ ERROR OVER ------------------------
################ Test zdtm/transition/ipc FAIL at CRIU restore #################
```

**Comment by avagin** [comment]
https://ci.openvz.org/job/CRIU/job/CRIU-ppc64le/branch/criu-dev/125/console

**Comment by xemul** [comment]
Cc @cyrillos

**Comment by cyrillos** [comment]
yup, once I manage to access the machine will work on the issue

**Comment by xemul** [comment]
Should have been fixed by f6cc277cada0ad61b3fc017bf213ee4301d8e87f
